src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -2086,7 +2086,7 @@ public sealed class NoArgumentsInput : CalibrateInputBase
         public sealed class FixedPlattInput : CalibrateInputBase
         {
             [Argument(ArgumentType.AtMostOnce, ShortName = "slope", HelpText = "The slope parameter of the calibration function 1 / (1 + exp(-slope * x + offset)", SortOrder = 1)]
-            public Double Slope = 1;
+            public Double Slope = -1;
```

Comment:
>Slope [](start = 26, length = 5)

`FixedPlattCalibratorTrainer.Arguments` also has a `Slope` field that defaults to 1.